lib/factories.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Bigben.Factories do
```

Comment:
The typical convention is to keep module names singular. Up to you on what you decide, but thought I'd mention it